src/interfaces/constants.js

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
I don't remember that code style requires empty lines at the beginning

Reply:
fixed